packages/test/test-service-load/testConfig.json

```diff
@@ -86,6 +86,20 @@
             "totalBlobCount": 300,
             "blobSize": 10240,
             "detachedBlobCount": 10
+        },
+        "binarySnapshotFormat": {
```

Comment:
this config will only be run by you, in that the intention? or do you want it to run in CI?

Reply:
I would like to run in CI.

Reply:
well default for binary snapshot should be disabled right now. at some point we'll switch it to on by default once we have confidence. if you want this on in ci, you need to add the config override in the ci profile

Reply:
for this profile, you could just always turn it on, so `"Fluid.Driver.Odsp.binaryFormatSnapshot": [true]` and in the ci profile do `"Fluid.Driver.Odsp.binaryFormatSnapshot": [true,false]`. then you can run this profile manually on sets binary snapshot, and in ci it will be a mix, which is right, as that is how it will roll out in production